packages/app/src/app/pages/Sandbox/Editor/Workspace/items/Server/index.tsx

```diff
@@ -58,7 +58,13 @@ export const Server = inject('store', 'signals')(
           })
         );
       }
-    }, [editor.currentSandbox.id, ports, sandbox.template, server.ports]);
+    }, [
+      editor.currentSandbox.id,
```

Comment:
This line can be removed I think, since you're not using it inside the effect. 🤔

Reply:
Good catch! DOne :)